extensions/arc/src/localizedConstants.ts

```diff
@@ -61,6 +61,8 @@ export const controllerEndpoint = localize('arc.controllerEndpoint', "Controller
 export const extensionName = localize('arc.extensionName', "Extension name");
 export const extensionsDescription = localize('arc.extensionsDescription', "PostgreSQL provides the ability to extend the functionality of your database by using extensions. Extensions allow for bundling multiple related SQL objects together in a single package that can be loaded or removed from your database with a single command. After being loaded in the database, extensions can function like built-in features.");
 export const extensionsFunction = localize('arc.extensionsFunction', "Some extensions must be loaded into PostgreSQL at startup time before they can be used. These preloaded extensions can be viewed and edited  below.");
+export const extensionsAddFunction = localize('arc.extensionsAddFunction', "Some extensions must be loaded into PostgreSQL at startup time before they can be used. To edit, type in comma separated list of valid extensions: citus, pgaudit, pgautofailover, pg_cron, pg_partman, plv8, postgis, postgis_raster, postgis_sfcgal, postgis_tiger_geocoder, tdigest.");
```

Comment:
Where did this list of extensions come from?  I think there are more modules built into postgres that can be pre loaded.

Reply:
That is taken from the output given by the cli

![image](https://user-images.githubusercontent.com/69922333/127563716-58b5a8cd-d0c4-4831-8058-5092b3b9a89c.png)

Reply:
Does the CLI throw an error if anything else is specified?

Reply:
I see.  That's a backend validation we're doing with the webhook.  I'm not sure the values are exactly correct.  But I think it makes sense to duplicate the validation in the UI.  In general I'm okay with duplicating logic in the UI if it makes the experience better.